pkgs/development/libraries/hyperscan/default.nix

```diff
@@ -9,25 +9,22 @@
 #         which is fixed 8.41 version requirement (nixpkgs have 8.42+, and
 #         I not see any reason (for now) to backport 8.41.
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "hyperscan";
-  version = "5.4.0";
+  version = "5.4.2";
 
   src = fetchFromGitHub {
     owner = "intel";
-    repo = pname;
-    sha256 = "sha256-AJAjaXVnGqIlMk+gb6lpTLUdZr8nxn2XSW4fj6j/cmk=";
-    rev = "v${version}";
+    repo = finalAttrs.pname;
```

Comment:
```suggestion
    repo = "hyperscan";
```

Reply:
Thanks, that makes sense. I can't see anything else doing it in nixpkgs, but would it make sense to have both `hash` and `version` overridable (e.g. from `finalAttrs`)? Having just `version` overridable means you could change the version - but that'd be fruitless without changing the hash too.  

Or is the intention here just to side-step `rec` without allowing other packages to directly specify custom versions, etc.?

Reply:
> Thanks, that makes sense. I can't see anything else doing it in nixpkgs, but would it make sense to have both hash and version overridable

I think it's therefore better to override `src` in that case.

> Or is the intention here just to side-step rec without allowing other packages to directly specify custom versions, etc.?

I think that's the reason yeah.

Reply:
Thanks for implementing all the changes!!!